### PR TITLE
[estuary] skin sync

### PR DESCRIPTION
**Pull request by phil65**
@ronie

### CODE REVIEW

File: addons/skin.estuary/1080i/DialogVideoInfo.xml [hunk 1]
```diff
@@ -46,488 +70,471 @@
...
-						<top>10</top>
-						<control type="image">
-							<top>0</top>
-							<width>264</width>
-							<height>317</height>
-							<texture>DefaultActor.png</texture>
-							<aspectratio aligny="center">scale</aspectratio>
-							<bordertexture border="20">overlays/shadow.png</bordertexture>
-							<bordersize>20</bordersize>
+				<control type="list" id="50">
```
Comment (ronie) [inline addons/skin.estuary/1080i/DialogVideoInfo.xml]: might be worth to use preloaditems / background="true" for actor thumbs?

File: addons/skin.estuary/addon.xml [hunk 2]
```diff
@@ -23,7 +23,6 @@
 		<summary lang="sk_SK">Vzhľad Estuary, vytvorili phil65 a Piers. (predvolený vzhľad pre Kodi)</summary>
 		<summary lang="sr_RS@latin">Maska Estuary autora phil65 i Piers. (Podrazumevana Kodi maska)</summary>
 		<summary lang="sv_SE">Estuary-skin av phil65 och Piers. (Kodi's standard-skin)</summary>
-		<summary lang="tr_TR">Estuary dış görünümü. phil65 ve Piers (Kodi'nin varsayılan dış görünümü)</summary>
```
Comment (ronie) [inline addons/skin.estuary/addon.xml]: i don't think this is intended?

**Comment by ronie** [comment]
good to go.

File: addons/skin.estuary/addon.xml (hunk 2 above)
Comment (phil65) [inline addons/skin.estuary/addon.xml]: true. :)

File: addons/skin.estuary/1080i/DialogVideoInfo.xml (hunk 1 above)
Comment (phil65) [inline addons/skin.estuary/1080i/DialogVideoInfo.xml]: will check again, i think intention was to reduce actor thumb loading times on first display. Not sure if it helps though.

**Comment by phil65** [comment]
took care of addon.xml comment.

**Pull request merged**